### PR TITLE
Chore[New Exp]: Charts for HTTP Chaos Status Code

### PR DESCRIPTION
**Pull request by avaakash**
Signed-off-by: avaakash <as86414@gmail.com>

This PR adds the charts for the HTTP Chaos Status Code experiment.

#### Experiment PR
- https://github.com/litmuschaos/litmus-go/pull/539

#### Docs PR
- https://github.com/litmuschaos/litmus/pull/3677

### CODE REVIEW

File: charts/generic/experiments.yaml [hunk 1]
```diff
@@ -1223,6 +1222,127 @@ spec:
...
+          - "deletecollection"
+    image: "litmuschaos/go-runner:latest"
+    imagePullPolicy: Always
+    args:
+    - -c
+    - ./experiments -name pod-http-status-code
+    command:
+    - /bin/bash
+    env:
+
```
Comment (neelanjan00) [inline charts/generic/experiments.yaml]: Can we add envs for AppNS, AppLabel, etc?

Comment (avaakash) [reply charts/generic/experiments.yaml]: The appns etc are provided in the chaosengine (not in chaosexperiment).

Comment (neelanjan00) [reply charts/generic/experiments.yaml]: I suppose we should keep them in the ChaosExperiment too?

Comment (avaakash) [reply charts/generic/experiments.yaml]: The `AppNs`, `AppLabel` and `AppKind` are provided under the `appInfo` and not in `components/env` as we provide other env variables. This is consistent with all other charts. If there is a benefit of adding it in experiments.yaml as well, I think we will need to discuss with maintainers about such change.

File: charts/generic/experiments.yaml [hunk 2]
```diff
@@ -2037,7 +2036,10 @@ spec:
       app.kubernetes.io/service-kill: "true"
       app.kubernetes.io/version: latest
 
+<<<<<<< HEAD
```
Comment (uditgaurav) [inline charts/generic/experiments.yaml]: Conflicts are not resolved properly

File: charts/generic/pod-http-status-code/experiment.yaml [hunk 3]
```diff
@@ -0,0 +1,141 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    it injects chaos inside the pod which modifies the status code of the response from the provided application server to desired status code provided by the user and reverts after a specified duration
```
Comment (uditgaurav) [inline charts/generic/pod-http-status-code/experiment.yaml]:
```suggestion
    It injects chaos inside the pod which modifies the status code of the response from the provided application server to desired status code provided by the user and reverts after a specified duration
```

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**